cmd/config.go

```diff
@@ -34,26 +33,12 @@ import (
 	"github.com/loadimpact/k6/stats/kafka"
 	"github.com/loadimpact/k6/stats/statsd/common"
 	"github.com/pkg/errors"
-	"github.com/shibukawa/configdir"
```

Comment:
Maybe we can remove it as a dependancy if we can remove the other place where it is used https://github.com/loadimpact/k6/blob/config-painful-testing/cmd/root.go#L103-L104  . Probably not for this issue but for an issue

Reply:
Okay now I saw your comment ;)

Reply:
https://github.com/loadimpact/k6/issues/940